$NetBSD$

These defines are set in the Makefile to create the different
versions of wolf4sdl, instead of building only one version.

--- version.h.orig	2015-04-17 01:23:41.000000000 +0000
+++ version.h
@@ -11,9 +11,9 @@
 
 //#define SPEAR
 //#define SPEARDEMO
-#define UPLOAD
-#define GOODTIMES
-#define CARMACIZED
+//#define UPLOAD
+//#define GOODTIMES
+//#define CARMACIZED
 //#define APOGEE_1_0
 //#define APOGEE_1_1
 //#define APOGEE_1_2
